### PR TITLE
Fix knative prow contrller manager indentation, add yamllint job

### PR DESCRIPTION
**Pull request by chaodaiG**

### CODE REVIEW

File: prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml [hunk 1]
```diff
@@ -20,6 +20,21 @@ presubmits:
             # Details: https://github.com/GoogleCloudPlatform/oss-test-infra/pull/131#discussion_r334208082
             - --exclude-warning=mismatched-tide
             - --exclude-warning=non-decorated-jobs
+  - name: pull-lint-valiaditor
```
Comment (hopkiw) [inline prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml:23]: spelling: validator?

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml [hunk 2]
```diff
@@ -96,8 +96,6 @@ postsubmits:
         - "/main.sh"
         args: ["gcp-guest", "us-west1-a", "deb9,el6,el7,el8,goo", "gs://gcp-guest-package-uploads/osconfig/"]
     max_concurrency: 10
-    branches:
```
Comment (hopkiw) [inline prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml:99]: can you explain why you've removed this?

Comment (chaodaiG) [reply prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml:99]: It's a duplicate, see a few lines above

File: prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml (hunk 1 above)
Comment (chaodaiG) [reply prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml:23]: Yeah, let me fix it

Comment (chaodaiG) [reply prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml]: Fixed, thanks!

**Review by chases2 — APPROVED** [review]

**Pull request merged**